src/04_printing.py

```diff
@@ -11,7 +11,8 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print('x is %d, y is %f, z is "%s"' % (10, 2.24552, "I like turtles!"))
 # Use the 'format' string method to print the same thing
-
-# Finally, print the same thing using an f-string
\ No newline at end of file
+print('x is {}, y is {}, z is "{}"'.format(10, 2.24552, "I like turtles!"))
+# Finally, print the same thing using an f-string
+print(f'x is {x}, y is {y}, z is "{z}"')
```

Comment:
It's pretty amazing how many string formatting tools that are available to you; I recommend choosing one and staying with it.